validation/connection_fields.xsd

```diff
@@ -25,7 +25,13 @@
     <xs:attribute name="default-value" type="xs:string"/>
     <xs:attribute name="editable" type="xs:boolean"/>
     <xs:attribute name="secure" type="xs:boolean"/>
-    <xs:attribute name="name" type="xs:string" use="required"/>
+    <xs:attribute name="name" use="required">
+     <xs:simpleType>
+        <xs:restriction base="xs:string">
+          <xs:pattern value="[a-zA-Z_]([a-zA-Z0-9_.-])*"/>
```

Comment:
```suggestion
          <xs:pattern value="[:a-zA-Z_]([a-zA-Z0-9_.-])*"/>
```
Add back `:` until we know more.  I want this to be able to match the monolith XSD as well.

Reply:
Adding `:` back.